gateway-js/src/__tests__/loadSupergraphSdlFromStorage.test.ts

```diff
@@ -380,20 +371,96 @@ describe('loadSupergraphSdlFromStorage', () => {
       );
     });
 
-    it("throws on timeout status response and successfully submits an out of band error", async () => {
+    it('Out of band reporting doesnt submit reports when endpoint is not configured', async () => {
+      mockSupergraphSdlRequest().reply(400);
```

Comment:
We might add a comment here to help illustrate how this test actually tests the thing. i.e.
```
// if an additional request were made by the out of band reporter, nock would throw since it's unmocked
// and this test would fail
```

Without this knowledge it could be unclear as to how this test is exercising our case at all.